util/reflect.go

```diff
@@ -1029,6 +1029,10 @@ func getNodesList(schema *yang.Entry, root interface{}, path *gpb.Path) ([]inter
 				if err != nil {
 					return nil, nil, err
 				}
+				// NOTE: Normally we'd like to use ygot.KeyValueAsString for conversion
```

Comment:
Is this true now we implemented the log string change? If there are two invalid values - do they end up being the same string representation?

Reply:
From below, I see that the invalid value is included in the error message, so I think this is fine.